### ISSUE TITLE
Mail support

### ISSUE THREAD

**Issue by cemo** [retrievercommunications/docker-jasperserver#5]
Is there an easy way to add support for email by environments variables? 
(By the way I am using in Kubernetes cluster your image. It is working pretty much nice.)

**Comment by grange74** [comment]
I assume that you mean configuring the mail server and from address?

From what i can see this could be done by creating 6 new env vars and using them to set the following 6 properties in the default_master.properties via the current "entrypoint.sh" similarly to how the database settings are set.

```
quartz.mail.sender.host=mail.example.com
quartz.mail.sender.port=25
quartz.mail.sender.protocol=smtp
quartz.mail.sender.username=admin
quartz.mail.sender.password=password
quartz.mail.sender.from=admin@example.com
```

**Comment by cemo** [comment]
Yes, this is what I need. [bitname docker image] has a feature like this as well. 

[bitname docker image]: https://github.com/bitnami/bitnami-docker-jasperreports

**Comment by grange74** [comment]
Ok, good. I will look at implementing that when i have some free time.

Btw that's an interesting image you reference. It seems newer and a bit more sophisticated than our one. Just wondering how come you aren't using that one instead?

Also I noticed that they have a data volume for the jasperserver container itself and i was trying to look at what it stores but i couldn't see as they seem to use some custom code that is probably in another repo to do the setup.

**Comment by cemo** [comment]
Actually I had to use an image which was ready to connect our database instances. I found pretty much is to configure your image at that time.

**Comment by nataliiakel** [comment]
Closing as an issue seems to be resolved.
